doc/source/overview.rst

```diff
@@ -117,6 +117,46 @@ Institutional Partners
 
 The information about current institutional partners can be found on `pandas website page <https://pandas.pydata.org/about.html>`__.
 
+Modules Privacy Has Changed
+----------------------------
+
+Some formerly public python/c/c++/cython extension modules have been moved and/or renamed. These are all removed from the public API. Furthermore, the pandas.core, pandas.compat, and pandas.util top-level modules are now considered to be PRIVATE. If indicated, a deprecation warning will be issued if you reference theses modules. (GH12588)
```

Comment:
Can you replace (GH12588) with (:issue:\`12588\`) here, and likewise with the other instances of this pattern elsewhere.  I don't think the GH pattern will actually link to the issues.

Reply:
Alright, I'll fix that.